src/app/pages/culture/culture.component.ts

```diff
@@ -1,103 +1,114 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit,OnDestroy } from '@angular/core';
+import { Router } from '@angular/router';
 import Card from 'src/app/models/card';
+import { CultureService,details } from 'src/app/services/culture-service.service';
```

Comment:
Wrong way,  service and interface imported from service

Reply:
changes this one too...imported services from service and interface from interface